### ISSUE TITLE
Backlinks button won't appear after hiding it or going to a note that has no backlinks.

### ISSUE THREAD

**Issue by XXXJetfireXXX** [zadam/trilium#3158]
### Trilium Version

0.55.1

### What operating system are you using?

Windows

### What is your setup?

Local (no sync)

### Operating System Version

Windows 10

### Description

As described in the title, backlinks button won't appear after hiding it or going to any other note that has no backlinks.
The only way I can make it appear again is by reloading frontend.

**Comment by zadam** [comment]
Hi, thanks for reporting the problem, fixed.

**Comment by xuxanwan** [comment]
> Hi, thanks for reporting the problem, fixed.

at 0.56.0-beta, problem still exists

**Comment by zadam** [comment]
@xuxanwan I can't reproduce that anymore. Could you post more details or maybe video?

**Comment by xuxanwan** [comment]
> @xuxanwan I can't reproduce that anymore. Could you post more details or maybe video?

You can watch the gif below：
![2022-10-20-18-00-58](https://user-images.githubusercontent.com/5035496/196919223-e9860442-9250-41a5-b5ce-544f26bea576.gif)
![2022-10-20-17-56-50](https://user-images.githubusercontent.com/5035496/196919316-ca81b615-5177-4237-ae25-7bc6e57d6751.gif)

**Comment by zadam** [comment]
@xuxanwan I see, unfortunately I can't reproduce it. Would it be possible to export the "aa" subtree in ZIP and send it to here or to zadam.apps@gmail.com? Thanks.

![image](https://user-images.githubusercontent.com/617641/197041919-0b20f830-5297-483c-9cd3-b0b3eaf88ba4.png)

BTW, is this a desktop version or web? If web, could you try a hard reload clearing the cache?

Is the behavior as described in the original ticket or does the backlink button just never appears?

**Comment by xuxanwan** [comment]
@zadam 
I've found this problem only occur whenever the notes are below the diary note.
![image](https://user-images.githubusercontent.com/5035496/197206684-c1a9b692-e412-4f24-9e22-8eb5fa9f75dc.png)
When I moved note outside, everything worked fine
![image](https://user-images.githubusercontent.com/5035496/197207527-0b5a17c1-cd01-4f40-b43f-90415ecbdf37.png)

And this is desktop version.

 I used 'ctrl +shift +r' to force refresh and the problem persists.
